src/main/java/dinkplugin/SettingsManager.java

```diff
@@ -70,16 +81,52 @@ public boolean isNamePermitted(String name) {
     @VisibleForTesting
     public void init() {
         setIgnoredNames(config.ignoredNames());
-        configManager.getConfigDescriptor(config).getItems()
-            .forEach(item -> configValueTypes.put(item.key(), item.getType()));
+        configManager.getConfigDescriptor(config).getItems().forEach(item -> {
+            String key = item.key();
+            configValueTypes.put(key, item.getType());
+
+            String section = item.getItem().section();
+            if (StringUtils.isNotEmpty(section)) {
+                keysBySection.computeIfAbsent(
+                    section.toLowerCase().replace(" ", ""),
+                    s -> new HashSet<>()
+                ).add(key);
+            }
+        });
     }
 
     void onCommand(CommandExecuted event) {
         String cmd = event.getCommand();
-        if ("dinkimport".equalsIgnoreCase(cmd)) {
+        if ("DinkImport".equalsIgnoreCase(cmd)) {
             importConfig();
-        } else if ("dinkexport".equalsIgnoreCase(cmd)) {
-            exportConfig();
+        } else if ("DinkExport".equalsIgnoreCase(cmd)) {
+            String[] args = event.getArguments();
+
+            Predicate<String> includeKey;
+            if (args == null || args.length == 0) {
+                includeKey = k -> !WEBHOOK_CONFIG_KEYS.contains(k);
+            } else {
+                includeKey = k -> false;
+
+                for (String arg : args) {
+                    if ("all".equalsIgnoreCase(arg)) {
```

Comment:
Can we add an option for "default" which is the same as sending no argument?

Reply:
Sure, as along as people don't interpret that as exporting the plugin's default values for each config item

Reply:
I do agree this will likely cause confusion that it's not the settings `default` value

Reply:
also maybe we don't need this until a later date where you can do default + specific url

right now:

* export default + notifier => just do export default
* export default => just do export without arg
* export default + webhooks => just do export all

Reply:
That's fine by me